cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -110,8 +130,10 @@ _kubectl label node $SRIOV_NODE sriov=true
 
 wait_pods_ready
 
-# we need to sleep as the configurations below need to appear
-sleep 30
+# Ensure webook-configuration object created
+ensure_cr "validatingwebhookconfiguration" "operator-webhook-config"
```

Comment:
None of these object types are custom, all are part of Kubernetes. Please rename the function to `ensure_object` or something more generic.

Reply:
ack